### PR TITLE
Add region to the dynamo db span

### PR DESCRIPTION
**Pull request by andriikushch**
This PR updates AWS SDK instrumentation. It extends DynamoDB span with a "region" tag.

### CODE REVIEW

File: instrumentation/instaawssdk/dynamodb_tags.go
Comment (basti1302) [inline instrumentation/instaawssdk/dynamodb_tags.go:59]: You could extract capturing the `dynamodbTable` tag also from the individual cases and put it here, only omitting it if type is "list tables". Just a thought.

**Review by basti1302 — APPROVED** [review]
LGTM

File: instrumentation/instaawssdk/dynamodb_tags.go
Comment (willianpc) [reply instrumentation/instaawssdk/dynamodb_tags.go:59]: +1 for this suggestion, although feel free to leave it to another time if you want, as it does not directly impact the change being done.

**Review by willianpc — APPROVED** [review]
LGTM.
My only comment is about Bastian's suggestion.

**Comment by willianpc** [comment]
> LGTM
> 
> (CircleCI is not happy though)

Just for clarity, this error happens because this PR depends on a new version (v1.44.0) of the tracer core.
Right now, this PR points to an unknown version of the tracer core. 
Once Andrii merges that PR and releases the aforementioned v1.44.0, this PR will then find the correct version - which it depends on :)

File: instrumentation/instaawssdk/dynamodb_tags.go
Comment (andriikushch) [reply instrumentation/instaawssdk/dynamodb_tags.go:59]: Good point 👍 

I played with code a little bit, but so far I do not have an idea how to do this without an extra if statement :(. In my opinion, having it doesn't make code easier to read.

Comment (basti1302) [reply instrumentation/instaawssdk/dynamodb_tags.go:59]: Yes, that would need an extra if statement. It was really just a thought to reduce the code duplication a bit. But I agree, it is totally a matter of taste, so feel free to ignore the suggestion entirely. 👍

Comment (andriikushch) [reply instrumentation/instaawssdk/dynamodb_tags.go:59]: 👍

**Pull request merged**
